CHANGELOG.md

```
@@ -3,6 +3,7 @@
 #### 9.4.18 - October 2024
   * Added new `FieldEnum` with all the available field names
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)